src/udev/udev-node.c

```diff
@@ -31,7 +31,7 @@ int udev_node_cleanup(void) {
         _cleanup_closedir_ DIR *dir = NULL;
 
         /* This must not be called when any workers exist. It would cause a race between mkdir() called
-         * by stack_directory_lock() and unlinkat() called by this. */
+         * by link_directory_lock() and unlinkat() called by this. */
```

Comment:
I agree with that `stack directory` is misleading, but `link directory` sounds ambiguous for me...

Reply:
`link_directory` sounds  "the directory for a given symlink", which seems ok to me. Otherwise I don't have a better name to suggest.

Reply:
OK. The naming is used only internally. Please go ahead.